site/data/events/osdc2019.yaml

```
@@ -0,0 +1,6 @@
+event: Open Source Datacentre Conference
```

```suggestion
event: Open Source Data Center Conference
```

Source: https://osdc.de